cli/lib/cli.js

```diff
@@ -114,6 +116,7 @@ module.exports = {
     .option('--group', text('group'), coerceFalse)
     .option('--group-id <group-id>', text('groupId'))
     .option('--dev', text('dev'), coerceFalse)
+    .option('--cypress-path <cypress path>', text('cypressPath'))
```

Comment:
probably want to say this is `cypress folder` otherwise it might seem like Cypress app filepath

Reply:
should we do `--cypress-dir` or `--cypress-folder`? or something longer like `--path-to-binary`